issues/user-attrs/README.md

```diff
@@ -0,0 +1,111 @@
+# Issue: User Attibutes
+
+In a production like scenario, roles have been created to restrict what the user sees in the OData services. In this example, the user is restricted to viewing certain authors in the fake bookstore.
+
+Some OData queries throw SQL errors (in SQLite and HANA) when they are valid OData queries and a response is expected.
+
+This example is based on DJ's computed-field sample and this (link)[https://cap.cloud.sap/docs/guides/authorization#user-attrs].
+
+Note: The documentation (here)[https://cap.cloud.sap/docs/node.js/authentication#authentication-strategies] states to use `userAttributes` in the `package.json` however, only `attributes` property works.
+
+
+## Try it out
+
+First, get the service up and running:
+
+1. Get to right place: `cd issues/user-attrs/`
+2. Install deps and run in memory (with debug): `npm i && DEbUG=true cds run --in-memory`
```

Comment:
With the spelling correction:

DEBUG=true

the debug output would work.

Reply:
Fixed in commit 82808b1